platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/rest/ApplicationUploadEndpoint.java

```diff
@@ -159,73 +155,65 @@ public Response create(MultipartBody multipartBody, @Context UriInfo requestUriI
    * @param attachment the attachment to copy and extract.
    * @return The file of the copied attachment.
    */
+  @Nullable
   private File createFileFromAttachement(Attachment attachment) {
-    InputStream inputStream = null;
-    String filename = null;
-    File newFile = null;
-    if (attachment.getContentDisposition() != null) {
-      filename =
-          attachment
-              .getContentDisposition()
-              .getParameter(FILENAME_CONTENT_DISPOSITION_PARAMETER_NAME);
-    }
-
-    if (StringUtils.isEmpty(filename)) {
-      LOGGER.debug("Filename not found, using default.");
-      filename = DEFAULT_FILE_NAME;
-    } else {
-      filename = FilenameUtils.getName(filename);
-      LOGGER.debug("Filename: {}", filename);
-    }
+    try (InputStream inputStream = attachment.getDataHandler().getInputStream()) {
```

Comment:
❓ This will now possibly throw an exception from the auto-close on the input stream whereas before it was ignored. Do we really want to do that? 
Closing exceptions on input streams should typically be swallowed.

Reply:
I agree. We may want to leave this as a traditional try-catch and suppress the SonarQube finding.